Tools/ardupilotwaf/boards.py

```diff
@@ -894,6 +894,9 @@ def configure_env(self, cfg, env):
             cfg.msg("Enabling ChibiOS asserts", "no")
 
 
+        if cfg.env.SAVE_TEMPS:
+            env.CXXFLAGS += [ '-S', '-save-temps=obj' ]
```

Comment:
Should this really be in this PR?  My understanding is it doesn't currently work...

Reply:
it does work, as in it can be used to find why the cache hit rate has dropped, it just fails the link stage, as g++ puts the assembler in the *.o object file, but you don't need to link for this to be useful